tests/regression/lp-1805485/task.yaml

```diff
@@ -0,0 +1,10 @@
+summary: apparmor backend is not used on openSUSE Leap 42.3
+details: |
+    AppArmor on openSUSE Leap 43.2 is too old to parse the snap-confine
+    apparmor profile.  Because of "relaxed" error handling snapd was enabling
+    apparmor backend even though it resulted in constant problems with setting
+    up security for the core snap. With this bug fixed the old version of
+    apparmor_parser makes snapd disable the relevant security backend. 
+systems: [opensuse-42.3-64]
```

Comment:
Shouldn't the test check the supported systems have not regressed after the fix was introduced?
```
systems: [ubuntu-*, opensuse-*, arch-linux-*]
execute:
    if [[ "$SPREAD_SYSTEM" == opensuse-42.3* ]]; then
        # openSUSE 43.2 has AppArmor parser in version too old to parse snap-confine profile
        snap debug sandbox-features | MATCH -v "^apparmor:"
    else
        snap debug sandbox-features | MATCH "^apparmor:"
    fi

Reply:
We have plenty of places that do that but I can add a safety check yeah,